components/data/data-services/org.wso2.micro.integrator.dataservices.core/pom.xml

```diff
@@ -233,6 +233,24 @@
             <groupId>org.apache.ws.security.wso2</groupId>
             <artifactId>wss4j</artifactId>
         </dependency>
+        <dependency>
+            <groupId>de.undercouch</groupId>
+            <artifactId>bson4jackson</artifactId>
+            <exclusions>
```

Comment:
We don't need the exclusions here as we have in parent pom ?

Reply:
Removed the exclusions.